### PR TITLE
Allow ppltasks.h and pplxtasks.h to co-exist

### PR DESCRIPTION
**Pull request by ioannis-e**

### CODE REVIEW

**Comment by ioannis-e** [comment]
@BillyONeal I also get a warning C4005: '_CAPTURE_CALLSTACK': macro redefinition when both ppltasks.h and pplxtasks.h are included.

Should i proceed with renaming the macro _CAPTURE_CALLSTACK to PPLX_CAPTURE_CALLSTACK in pplxtasks.h to resolve the warning ?

File: Release/include/pplx/pplxtasks.h [hunk 1]
```diff
@@ -1460,7 +1464,7 @@ namespace details
         virtual ~_ContinuationTaskHandleBase() {}
     };
 
-#if _PPLTASK_ASYNC_LOGGING
+#if _PPLTASK_ASYNC_LOGGING && _MSC_VER >= 1800 && defined(__cplusplus_winrt)
```
Comment (BillyONeal) [inline Release/include/pplx/pplxtasks.h]: Hmm it isn't clear why this test isn't merged into _PPLTASK_ASYNC_LOGGING as it was before?

File: Release/include/pplx/pplxtasks.h [hunk 2]
```diff
@@ -136,13 +136,17 @@ namespace std
     }
 }
 #endif /* _MSC_VER < 1700 */
-#ifndef _PPLTASK_ASYNC_LOGGING
```
Comment (BillyONeal) [inline Release/include/pplx/pplxtasks.h]: It seems like a better change would be to avoid this mess by just renaming this to PPLX_TASK_ASYNC_LOGGING ?

**Comment by ioannis-e** [comment]
I agree and have made the relevant renames.

There is a small mistake in the commit message, which should have read:
1. Rename _PPLXTASKS_H to PPLXTASKS_H

**Pull request merged**